client/packages/invoices/src/Returns/modals/InboundReturn/InboundReturn.tsx

```diff
@@ -57,6 +59,10 @@ export const InboundReturnEditModal = ({
     itemId,
   });
 
+  useEffect(() => {
+    if (initialItemId) setItemId(initialItemId);
+  }, [initialItemId]);
+
```

Comment:
I could avoid adding the `useEffect` if I also called `setItem` as part of `handleNextItem`, but this seems more 
"correct" to me, as one state should cause the other to update rather than updating it in two places.